src/server.js

```diff
@@ -1,43 +1,46 @@
-var express = require('express');
-var itc = require("itunesconnect");
-var auth = require('basic-auth');
+"use strict";
+
+var express = require('express'),
+    itc = require("itunesconnect"),
+    auth = require('basic-auth');
 
 var app = express();
 var Report = itc.Report;
 
 // Auth
-app.all('*', function(req, res, next) {
-   var user = auth(req);
-
-   if (!user) {
-      res.set('WWW-Authenticate', 'Basic realm=Authorization Required');
-      return res.sendStatus(401);
-   }
-
-   var options = {
-      errorCallback: function(err) {
-         res.set('WWW-Authenticate', 'Basic realm=Authorization Required');
-         return res.sendStatus(401);
-      },
-      loginCallback: function(resp) {
-      }
-   }
-
-   var itunes = new itc.Connect(user.name, user.pass, options);
-   req.itunes = itunes;
-
-   next();
+app.all('*', function (req, res, next) {
```

Comment:
I'm pretty sure that you can do this with [middleware](http://expressjs.com/guide/using-middleware.html) and with `use` function.